javav2/example_code/.samplefiles/sample_private_key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I can not see where this is used, but just be sure that you are not exposing a secret.